### ISSUE TITLE
Enum value validation

### ISSUE THREAD

**Issue by LukasWieditz** [rebing/graphql-laravel#812]
### Versions:
- graphql-laravel Version: 7.2.0
- Laravel Version: 8.53.1
- PHP Version: 8.0.0

### Description:

When creating an enum type with an invalid value according to the [GraphQL spec](https://spec.graphql.org/June2018/#sec-Names), validation falsely passes without errors.

### Steps To Reproduce:

1. Create the following sample enum:
```php
class MyEnum extends Enum
{
    protected $enumObject = true;

    protected $attributes = [
        'name' => 'MyEnum',
        'description' => 'An enum',
        'values' => ['1', 'Half', 'OT', 'F', 'F/OT'],
    ];
}
```

2. Schema validation will falsely pass, failing to identify `1` & `F/OT` as invalid enum values.
3. Consumer libraries will fail to validate the regex in the above linked spec.

**Comment by mfn** [comment]
I _think_ this behavior is part of graphql-php, the library this one is built upon.

A few times already I managed to break the schema, until I figured I need a separate validation.

For performance reason, also in graphql-php, there's no automatic validation of the schema; it has to be called manually -> see https://github.com/webonyx/graphql-php/blob/c886d2bf0679f6dc010209c2e64961776c74bd27/src/Type/Schema.php#L579

What I did in private projects is I created an artisan command to validate the schema and run it as part of CI:
```php
        $schema = app('graphql')->schema($schemaName);
        $result = $schema->validate();

        if ($result) {
            $this->error("Errors have been detected in the given GraphQL schema '$schemaName':");
            dump($result);
        }
```
I always meant add a artisan command like this to graphql-laravel but never got around; it won't automatically validate the schema but at least would give you ready-made tool for CI.
